docs/2_1_common_api.adoc

```diff
@@ -203,27 +203,38 @@ this function has shown the prepared information message to the user.
 
 `fmi3Discard`::
 [[fmi3Discard,`fmi3Discard`]]
+With this return value, the FMU signals that the function called could not achieve the defined effect and was able to keep its state as before the call.
+This essentially indicates that the function call was a NOOP.
```

Comment:
I had to google NOOP,
Maybe I who is un educated since I lack a CS background with very limited assembler experience.
Is it a commonly used expression or should it be changed?